package/MDAnalysis/analysis/encore/clustering/Cluster.py

```diff
@@ -0,0 +1,212 @@
+# Cluster.py --- classes to handle results of clustering runs
+# Copyright (C) 2014 Wouter Boomsma, Matteo Tiberti
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Ensemble representation --- :mod:`MDAnalysis.analysis.ensemble.ensemble`
+=====================================================================
+
+The module contains the Cluster and ClusterCollection classes which are
```

Comment:
I like the idea and you could make the argument that this could eventually become `MDAnalysis.lib.cluster`, assuming that it is free of dependencies that would prevent this.

However, for right now, I am inclined to leave it in encore and make sure that encore works and can go into 0.15.0. We can then open an issue to move clusters out of encore but this won't break anything so we can do it at our leisure. 

I want to avoid heaping even more tasks on @wouterboomsma and @mtiberti who have been already doing _a lot_.